### PR TITLE
Use the cocina DRO types

### PR DESCRIPTION
**Pull request by jcoyne**
Ref  https://github.com/sul-dlss/dor-services-client/issues/106

### CODE REVIEW

File: lib/cocina/models/dro.rb [hunk 1]
```diff
@@ -6,6 +6,23 @@ module Cocina
   module Models
     # A digital repository object.  See https://github.com/sul-dlss-labs/taco/blob/master/maps/DRO.json
     class DRO < Dry::Struct
+      DRO_TYPES = %w[
```
Comment (mjgiarlo) [inline lib/cocina/models/dro.rb]: Would you consider naming this TYPES since it's already in the DRO context?

Comment (jcoyne) [reply lib/cocina/models/dro.rb]: yep, changed

File: lib/cocina/models/dro.rb [hunk 2]
```diff
@@ -6,6 +6,23 @@ module Cocina
...
+        http://sdr.sul.stanford.edu/models/sdr3-geo.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-image.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-page.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-photograph.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-manuscript.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-map.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-media.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-track.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-webarchive-binary.jsonld
+        http://sdr.sul.stanford.edu/models/sdr3-webarchive-seed.jsonld
```
Comment (mjgiarlo) [inline lib/cocina/models/dro.rb]: I'm not super stoked about keeping the `sdr3` name around, but I realize that making that change *here* is not the right place. (And I don't know how much support we have, nor what process we'd use, for tweaking the modeling documentation.) Just registering that with you to see what you think. Approving PR.

Comment (jcoyne) [reply lib/cocina/models/dro.rb]: I'd be okay with me if we drop the `3`

**Pull request merged**